src/main/cli/settings.c

```diff
@@ -1215,6 +1215,7 @@ const clivalue_t valueTable[] = {
 #endif
     { "osd_cap_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 20000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, cap_alarm) },
     { "osd_alt_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 10000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, alt_alarm) },
+    { "osd_distance_alarm",         VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 0, UINT16_MAX }, PG_OSD_CONFIG, offsetof(osdConfig_t, distance_alarm) },
```

Comment:
What is the unit for this? I know it's not consistent, but wouldn't it be better to add the unit to the name, like `osd_distance_alarm_m`?

Reply:
The units are meters. The name is consistent with the existing altitude alarm `osd_alt_alarm`.

Reply:
Consistent - yes. Good - not sure, I think adding units for new parameters would be better.